ppa/archive/tests/test_views.py

```diff
@@ -1175,16 +1176,18 @@ def test_get_pages(self):
 
             mock_queryset_cls.assert_called_with()
             mock_qs = mock_queryset_cls.return_value
-            mock_qs.search.assert_any_call(content="(iambic)")
+
+            ## These asserts aren't working ###
+            mock_qs.filter.assert_any_call(group_id__in=['"work1"', '"work2"'])
             mock_qs.filter.assert_any_call(item_type="page")
-            mock_qs.filter.assert_any_call(group_id_s__in=['"work1"', '"work2"'])
-            mock_qs.group.assert_called_with("group_id_s", limit=2, sort="score desc")
+            mock_qs.search.assert_any_call(content="(iambic)")
+            mock_qs.group.assert_called_with("group_id", limit=2, sort="score desc")
             mock_qs.highlight.assert_called_with(
                 "content", snippets=3, method="unified"
             )
             mock_qs.get_response.assert_called_with(rows=100)
-
             assert highlights == mock_qs.get_highlighting()
+            ####################################
```

Comment:
@rlskoeser when you get a sec could you look at these lines? These are the last lines failing tests. It's clearly due to changing the SolrQuerySet -> PageSearchQuerySet as the mock class, but any idea why the assert_any_call or assert_called_with are breaking now?